apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -244,6 +244,8 @@ public void startTestUsingBundle() {
             mAudioOutTester.setSignalType(signalType);
 
             openAudio();
+            int burstCount = IntentBasedTestSupport.getBurstCount(mBundleFromIntent);
```

Comment:
just to confirm, this was missing before even though our documentation says it was supported for output tests right?

Reply:
I only saw it was used in glitch test and roud trip latency test. https://github.com/search?q=repo%3Agoogle%2Foboe%20KEY_BUFFER_BURSTS&type=code